src/app/shared/models/storage.model.ts

```diff
@@ -0,0 +1,5 @@
+export enum LocalStorage {
```

Comment:
LocalStorageKeys ? localstorage is native and we should avoid similar names

Reply:
Zgadzam się. Wystarczy już, że backend nam generuje typ `Location`, co konfliktuje z `window.location`...